docs/en/04_Changelogs/4.4.0.md

```diff
@@ -120,21 +126,26 @@ this can lead to improved search engine rankings (since existing files under new
 
 #### Can I migrate away from the legacy_filenames=true option?
 
-Technically yes, but there’s no official migration script for it.
+Yes. Simply disable `legacy_filenames` in your YML configuration, then run the `MigrateFileTask`. This will normalise 
+the path of all existing assets.
 
 #### Can I still choose legacy_filenames=true when starting new upgrades?
 
-Technically yes, but you need to be aware of the tradeoffs.
-Once the regression has been fixed, we don’t see the need for people to choose this option any more.
+Technically yes, but you need to be aware of the tradeoffs. Once the regression has been fixed, we don’t see the need 
```

Comment:
If that second part is true, the first part is redundant (there are no tradeoffs?)